build/styles/style.css

```diff
@@ -55,10 +55,7 @@ li {
 
 #form {
   float: right;
-  width: 300px; }
-
-#alert {
-  width: 100px; }
+  width: 200px; }
```

Comment:
Hey, nice work.  I'd push the closing curly to the next line for best practices.  Eventually we'll use linters to help you ramp up on the accepted norms for syntax.  Reference http://codeguide.co/.  This guy wrote Bootstrap, and much of github, checkout his guide.